samples/file-encryptor/enclave/mbedtls_src/keys.cpp

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 #include <mbedtls/aes.h>
-#include <mbedtls/config.h>
+#include <mbedtls/mbedtls_config.h>
```

Comment:
Can you point out where does this change come from? I didn't see that in the current mbedtls version OE is using.

Reply:
See their commit to mbedtls [here](https://github.com/ARMmbed/mbedtls/commit/bb0cfeb2d4b34eb3ec932ca8b8e215f49b703153#diff-0279e5915ac49daea5fd9ed14934a074d1a88e9b35c674d7d1a0d0e0688f60bb).

Reply:
This change is in mbedtls 3.0.0 while OE currently uses 2.28
https://github.com/openenclave/openenclave/blob/65273c3117553139d67f03e21c1fb2adcb2d5d78/.gitmodules#L26

Reply:
Aha, maybe then we should just close this

Reply:
although I think OE could use 3.0 easily